controllers/commonservice_controller.go

```diff
@@ -181,6 +182,21 @@ func (r *CommonServiceReconciler) ReconcileMasterCR(ctx context.Context, instanc
 		}
 	}
 
+	for {
```

Comment:
I am wondering if we could remove the for loop here.

The main reason if that we would like to return error in CR reconciliation. Subsequently, it will trigger the next reconciliation, and shows `Failed` status in CommonService CR.